widgetssdk/src/main/res/values/strings.xml

```diff
@@ -43,7 +43,8 @@
     <string name="message_center_screen_title">Messaging</string>
     <string name="glia_dialog_end_engagement_title">End Engagement?</string>
     <string name="glia_chat_visitor_status_transferring">Transferring</string>
-    <string name="glia_dialog_screen_sharing_offer_title">You are about to share your screen</string>
+    <string name="glia_dialog_screen_sharing_offer_title">Start Screen Sharing</string>
+    <string name="glia_dialog_screen_sharing_offer_message">%1$s would like to see the screen of your device.</string>
```

Comment:
Based on the screenshots in the description, shouldn't it be like this?

```suggestion
    <string name="glia_dialog_screen_sharing_offer_message">%1$s has asked you to share your screen.</string>
```

Reply:
Oh, it seems that I have done the screenshot after enabling the remote locales back. But anyways I think it is right to have locally same strings as what we have in remote locales.
Thanks